tests/mini_sbibm/base_task.py

```diff
@@ -0,0 +1,119 @@
+import os
```

Comment:
please add license note to all new files. 

also, I think it would makes sense to add a note here that this was inspired by the sbibm-package.